src/wallet/wallet.ts

```diff
@@ -1099,6 +1099,57 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
     return parseInt(parts[5], 10);
   }
 
+
+  /**
+   * Get mint authorities
+   * Uses the getUtxos method to return one or many mint authorities
+   *
+   * @param tokenId of the token to select the authority utxo
+   * @param options Object with custom options.
+   *  {
+   *    'many': if should return many utxos or just one (default false),
+   *    'skipSpent': if should not include spent utxos (default true)
+   *  }
+   *
+   * @return Array of objects with {tx_id, index, address} of the authority output. Returns null in case there are no utxos for this type
+   **/
+  async getMintAuthority(tokenId: string, options: { many?: boolean, skipSpent?: boolean } = {}) {
```

Comment:
Missing return type

Reply:
Normalized the return of both facades on 3bc7cb238f85a5f9edb650477382f4e02c1f4eac, can you please check again?